### PR TITLE
Reorganize `static` folder and add pre-loaded stylesheets

### PR DESCRIPTION
**Pull request by edan-bainglass**
This PR reorganizes the static folder with dedicated subfolders for templates and styles. The styles folder holds stylesheets that are automatically loaded (using a new `aiidalab-widgets-base` utility function) in the main notebook.

## Dependency

This PR depends on https://github.com/aiidalab/aiidalab-widgets-base/pull/624, which introduces the `load_css` utility

### CODE REVIEW

**Comment by edan-bainglass** [comment]
@danielhollas, @superstar54, @yakutovicha the silicon select and confirm test is timing out. I tried to commit an increase to the timeout (to 60s), but it still is not enough. I'm surprised the new CSS loading or the reorganization of static files could cause such a delay. Could someone please give it a look?

**Comment by edan-bainglass** [comment]
Resolved. Thanks @superstar54 🙂

File: qe.ipynb [hunk 1]
```diff
@@ -24,6 +24,17 @@
...
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiidalab_widgets_base.utils.loaders import load_css\n",
+    "\n",
+    "load_css(css_path=\"src/aiidalab_qe/app/static/styles\")"
```
Comment (danielhollas) [inline qe.ipynb:35]: Can you use the `files` function here as well to make this consistent with the code below?

File: src/aiidalab_qe/app/static/styles/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Stylesheets for the Quantum ESPRESSO app
+
+This folder contains `css` stylesheets which may be loaded directly from the CSS folder using
+
+```python
+from aiidalab_widgets_base.utils.loaders import load_css_stylesheet
+
+load_css_stylesheet(package="aiidalab_qe.app.static.styles.css", filename="<stylesheet-name>.css")
```
Comment (danielhollas) [inline src/aiidalab_qe/app/static/styles/README.md:8]: I believe this README needs to be updated

File: src/aiidalab_qe/app/static/styles/custom.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.output_subarea {
+  max-width: none !important;
```
Comment (danielhollas) [inline src/aiidalab_qe/app/static/styles/custom.css:2]: Could you add a comment to this file explaining what this does?

**Review by danielhollas — COMMENTED** [review]
Looks reasonable, only one comment about the outdated README file. Leaving approval to others.

File: src/aiidalab_qe/app/static/styles/README.md (hunk 2 above)
Comment (edan-bainglass) [reply src/aiidalab_qe/app/static/styles/README.md:8]: Sure does 👍

File: qe.ipynb (hunk 1 above)
Comment (edan-bainglass) [reply qe.ipynb:35]: #751 completely restructures the notebook, moving the use of `files` internally to an app wrapper class.

**Review by danielhollas — APPROVED** [review]

**Pull request merged**